deployment-aws/terraform/cloudwatch.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_cloudwatch_log_group" "default" {
+
+  name = local.common_tags.Name
+
+  tags = {
+    Name = "${local.common_tags.Name}-cloudwatch"
+  }
```

Comment:
Tags are propagated from the provider, so this should not be necessary. Also, this resource is only used in the configuration of the API server, so I'd add it to `apiserver.tf`.

Reply:
I would also like the cromwell server to dump logs to cloudwatch, but haven't thought about that yet.